### PR TITLE
Add minimum database version check

### PR DESCRIPTION
**Pull request by Jellyfrog**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Required by #12580

File: LibreNMS/Validations/Database.php [hunk 1]
```diff
@@ -77,6 +87,30 @@ public function validate(Validator $validator)
...
+    {
+        $version = DB::selectOne('SELECT VERSION() as version')->version;
+        $version = explode('-', $version);
+
+        if (isset($version[1]) && $version[1] == 'MariaDB') {
+            if (version_compare($version[0], self::MARIADB_MIN_VERSION, '<=')) {
+                $validator->warn(
+                    'MariaDB version ' . self::MARIADB_MIN_VERSION . ' is the minimum supported version as of ' .
+                    self::MARIADB_MIN_VERSION_DATE . '. We recommend you update MariaDB to a supported version ' .
+                    self::MARIADB_RECOMMENDED_VERSION . ' suggested). If you do not update MariaDB, you will sooner or later run into problems.'
```
Comment (Jellyfrog) [inline LibreNMS/Validations/Database.php:100]: Different wording here?

Comment (SourceDoctor) [reply LibreNMS/Validations/Database.php:100]: good idea to check this.
why not a bool 'old_version' and point database type, version, and what is needed to variables
and after version_checks only one resulting string :-)

```
if (old_version): {
    $database_type . 'Version is ' . $running_version . '. Minimum Version is ' . $min_version . '. we recommend you to update your Database Server to at least Version ' . $recommended_version
}
```

Comment (murrant) [reply LibreNMS/Validations/Database.php:100]: If you do not update MariaDB, you will sooner or later run into problems. -> Failure to update will eventually cause issues.

**Review by murrant — APPROVED** [review]

**Pull request merged**